app.js

```diff
@@ -506,256 +506,148 @@ VIEWER.consumeForGeoJSON = async function(dataURL) {
             }
             return prev.concat(curr)
         }, [])
-        
 
-        /**
-         * Below this is helping people who did not put their properties in the Features.
-         * It will help along a Manifest or Canvas with navPlaces devoid of properties.
-         * Imagine being able to delete all this code if people just did their own properties!
-         * 
-         * TODO -- Too much C&P in this switch.  This should be broken down into helper functions.
-         */ 
-        switch(resourceType){
-            // This logic is the same as the Manifest logic, except looped over Collection.items which are Manifests.
-            case "Collection":
-                let coll_geos = []
-                if (VIEWER.resource.hasOwnProperty("navPlace")) {
-                    if (VIEWER.resource.navPlace.features) {
-                        VIEWER.resource.navPlace.features = VIEWER.resource.navPlace.features.map(f => {
+        /*
+        * Get the visible properties of the resource such as the thumbnail, description, and label.
```

Comment:
I'd like to see this documentation a little more fleshed out.  What makes the logic of this different from getManifestProperties()?